<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogevent</class>
 <widget class="QDialog" name="Dialogevent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion état Civile</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidge_etat">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>991</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_list">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_etat">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>911</width>
       <height>271</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prénom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>sexe</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date de naissance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>lieu de naissance</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>adresse mail</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>telephone</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>AKT23</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Hanfi</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>Nardine</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>Femme</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="5">
      <property name="text">
       <string>Tunis</string>
      </property>
     </item>
     <item row="0" column="6">
      <property name="text">
       <string>nardine.hanfi@esprit.tn</string>
      </property>
     </item>
     <item row="0" column="7">
      <property name="text">
       <string>94212389</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Menu">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>70</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_add">
    <widget class="QPushButton" name="pushButton_save">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>sauvgarder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>annuler</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelnom">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date de naisaance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lieu de naissance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>sexe</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>adresse mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_nom">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_prenom">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_sexe">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_mail">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_lieu">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>téléphone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_tel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>310</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifiant unique</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_id">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_naissance">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>330</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Retour </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="out_lieu">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>sexe</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="out_tel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lieu de naissance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>adresse mail</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="out_prenom">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>320</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>téléphone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="out_sexe">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date de naisaance</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelnom_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="out_naissance">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="out_mail">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>270</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="out_nom">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_maj_cit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>330</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_del_cit">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>330</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_impression">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>40</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>imprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>identifiant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="out_id">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="maj_save">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>300</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>sauvgarder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="maj_cancel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>300</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>annuler</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maj_nom">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>sexe</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maj_lieu">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lieu de naissance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>adresse mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maj_tel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>téléphone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date de naisaance</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maj_sexe">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="maj_mail">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="maj_prenom">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="maj_naissance">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelnom_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
